iron-autogrow-textarea.html

```diff
@@ -125,6 +134,50 @@
       },
 
       /**
+       * Bound to the textarea's `autocomplete` attribute.
+       */
+      autocomplete: {
```

Comment:
I think these are already defined in `PaperInputBehavior`? (which `paper-textarea` uses)